pkg/controller/controller.go

```diff
@@ -188,17 +191,26 @@ func sendEvent(obj, oldObj interface{}, c *config.Config, notifiers []notify.Not
 		}
 	}
 
-	// check for siginificant Update Events in objects
+	// Check for significant Update Events in objects
 	if eventType == config.UpdateEvent {
 		var updateMsg string
 		// Check if all namespaces allowed
-		updateSetting, exist := utils.AllowedUpdateEventsMap[utils.KindNS{Resource: kind, Namespace: "all"}]
+		updateSetting, exist := utils.AllowedUpdateEventsMap[utils.KindNS{Resource: resource, Namespace: "all"}]
 		if !exist {
 			// Check if specified namespace is allowed
-			updateSetting, exist = utils.AllowedUpdateEventsMap[utils.KindNS{Resource: kind, Namespace: objectMeta.Namespace}]
+			updateSetting, exist = utils.AllowedUpdateEventsMap[utils.KindNS{Resource: resource, Namespace: objectMeta.Namespace}]
 		}
 		if exist {
-			updateMsg = utils.Diff(oldObj, obj, updateSetting)
+			// Calculate object diff as per the updateSettings
+			var x, y *unstructured.Unstructured
```

Comment:
Can you please give proper variable names according to usage ?

Reply:
Done!